docs/infra/deploy_demo.md

```diff
@@ -0,0 +1,98 @@
+# Deploy to demo.eclipse-pass.org
+
+Our [demo.eclipse-pass.org](https://demo.eclipse-pass.org) system
+is currently not publically available.
+
+## Deploy
+
+From the [pass-docker](https://github.com/eclipse-pass/pass-docker) project
+you can view the available [actions](https://github.com/eclipse-pass/pass-docker/actions)
+including the [Deploy passdemo action](https://github.com/eclipse-pass/pass-docker/blob/main/.github/workflows/deploy_passdemo.yml)
+
+![pass-docker actions](/docs/assets/demo/passdocker_actions.png)
+
+You can then [run the workflow](https://github.com/eclipse-pass/pass-docker/actions/workflows/deploy_passdemo.yml)
+
+![run workflow](/docs/assets/demo/run_workflow.png)
+
+And watch the deploy.
+
+![deployed actions](/docs/assets/demo/deploy_actions.png)
+
+## Demo Code
+
+The code is located in an (unfortunately) deeply nested structure at
+
+```bash
+/opt/githubrunner/pass-docker/pass-docker/pass-docker
+```
+
+The deploy will pull the latest code, as shown below
+
+```bash
+a2forward@nightly-pass:/opt/githubrunner/pass-docker/pass-docker/pass-docker$ git log -1
```

Comment:
This was updated to:
``` sh
commit cc0b16359bf81ea1880e00e255a645aa6b090e75 (HEAD -> main, origin/main, origin/HEAD)
Author: John Abrahams <jabrah20@jhu.edu>
Date:   Mon Sep 26 10:21:08 2022 -0400

    Elide Demo Environment (#270)

    * Initial attempt at demo setup. Sort of working.

    * Add /authenticated route for Ember's auth to work
    ...
```

Not sure if we necessarily want this exact info here as it will change often. Unless we pin the demo version or only update it periodically, it may be burdensome to keep this bit of the doc up-to-date

Reply:
This was more to show "where" the code is located on the server, not necessarily needs to be the latest-latest, just showing "if you CD into this dir, you have fully access to git commands"